modules/backend/assets/vendor/jcrop/js/jquery.Jcrop.js

```diff
@@ -1084,10 +1081,9 @@
       //}}}
 
       // This is a hack for iOS5 to support drag/move touch functionality
-      // Hack OctoberCMS - the event handler was moved to the Touch module.
-      // The closure used before was handling a reference to the target object,
-      // preventing it from removing from DOM after the control is destroyed.
-      $(document).bind('touchstart.jcrop-ios', Touch.fixTouchSupport);
```

Comment:
Need to discuss this line

Reply:
I think we should revert this part or find why it was introduced as a `fix` :)

Reply:
Agree

Reply:
Added commit: https://github.com/octobercms/october/pull/4281/commits/448f543ac8236aaec1d04f68b1d8cdeb045b6fd8